src/autotype/AutoType.cpp

```diff
@@ -342,6 +355,13 @@ QList<AutoTypeAction*> AutoType::createActionFromTemplate(const QString& tmpl, c
     int num = -1;
     QList<AutoTypeAction*> list;
 
+    QRegExp delayRegEx("delay=(\\d+)", Qt::CaseSensitive, QRegExp::RegExp2);
```

Comment:
Delay is written lowercase and the flag is Qt::CaseSensitive. This mean that in the autotype string it must be lowercase so {delay=10} will pass and {DELAY=10} won't

Am I wrong? I haven't tested it because I'm on mobile but to me seems like an error